docs/submitting.md

```diff
@@ -220,6 +220,12 @@ Submission is as simple as:
 
 There are no fees for submitting or publishing in JOSS. You can read more about our [cost and sustainability model](http://joss.theoj.org/about#costs).
 
+## Preprint Policy
+
+Authors are welcome to submit their papers to a preprint server such as [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/) at any point during the submission and review process.
```

Comment:
Authors are welcome to submit their papers to a preprint server (e.g., [arXiv](https://arxiv.org/), [bioRxiv](https://www.biorxiv.org/), [SocArXiv](https://socopen.org/), [PsyArXiv](https://psyarxiv.com/)) at any point during the submission and review process.